<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1166</width>
    <height>827</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>771</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="felay_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Выбор режима выполнения</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dekay_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>С задержкой</string>
     </property>
    </widget>
    <widget class="QPushButton" name="without_delay_pushButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Без задержки</string>
     </property>
    </widget>
    <widget class="QPushButton" name="color_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>630</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Выбор цвета</string>
     </property>
    </widget>
    <widget class="QLabel" name="color_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>670</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="time_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>700</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Замер времени</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>740</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTableWidget" name="points_tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>251</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Введенные точки и ребра</string>
     </property>
    </widget>
    <widget class="QLabel" name="x_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>480</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;X:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="y_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>55</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Y:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="x_lineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>480</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="y_lineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>510</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>550</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить точку</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>590</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Замкнуть фигуру</string>
     </property>
    </widget>
    <widget class="QLabel" name="time_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>740</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Время построения:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>871</width>
      <height>771</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1166</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="program_menu">
    <property name="title">
     <string>О программе</string>
    </property>
   </widget>
   <widget class="QMenu" name="author_menu">
    <property name="title">
     <string>Об авторе</string>
    </property>
   </widget>
   <widget class="QMenu" name="clear_menu">
    <property name="title">
     <string>Очистка</string>
    </property>
   </widget>
   <widget class="QMenu" name="exit_menu">
    <property name="title">
     <string>Выход</string>
    </property>
   </widget>
   <addaction name="program_menu"/>
   <addaction name="author_menu"/>
   <addaction name="clear_menu"/>
   <addaction name="exit_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
